tests/users/test_api_user.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from flask import json
+
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1
+
+
+class TestResendEmailApi(BaseTestCase):
```

Comment:
can you also change the file name, as it is still `test_api_user.py` to `test_api_resend_email.py` @satya7289